src/Header/Nav/Nav.module.scss

```diff
@@ -62,22 +98,73 @@ $colour-nav-text: $colour-text-inverse;
       background: $colour-nice-white; // TODO: Update to $colour-body when the Design System is used everywhere
       color: $colour-text;
       font-weight: bold;
-      margin: rem($spacing-x-small 0);
+      //margin: rem($spacing-x-small 0);
 
       &:focus {
         outline-color: $colour-focus;
         outline-offset: rem(-$spacing-x-small);
       }
     }
   }
+}
 
-  abbr[title] {
-    border-bottom: 0;
-    cursor: help;
-    text-decoration: none;
+.subMenuWrapper {
+  background: #fff;
+  color: black;
+  display: block;
+}
+
+.subMenuList {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  a {
+    color: $colour-text;
+    padding-left: rem($spacing-large);
+
+    &:focus {
+      color: $colour-text;
+      outline: rem($spacing-x-small) solid $colour-focus;
+    }
+
+    &:visited {
+      color: $colour-text;
+    }
   }
+}
+
+@include mq($from: md) {
+  // TODO: Can we not use a magic number?
+  $sub-nav-height: 58;
+
+  .wrapperWithSubLinks {
+    padding-bottom: rem($sub-nav-height);
+  }
+
+  .menuList {
+    // Fallback for non flexbox browsers that support media queries e.g. IE9
+    display: table;
+    max-width: rem($nice-container-max-width);
+    // TODO Replace with mixin from nice design system
+    width: 96%;
```

Comment:
We seem to have two width properties here, is it valid to have two?

Reply:
width: 96%; and width: calc(100% - #{rem($spacing-large)});

Reply:
Yes, `width: 96%` is the fallback for browsers that don't support calc (IE8) see https://caniuse.com/#feat=calc for support